src/entity/user/experiences/UserExperience.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  Column,
+  Entity,
+  JoinColumn,
+  JoinTable,
+  ManyToMany,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+  TableInheritance,
+} from 'typeorm';
+import { User } from '../User';
+import { UserSkill } from '../UserSkill';
+import { ExperienceStatus, UserExperienceType } from './types';
+
+@Entity()
+@TableInheritance({ column: { type: 'text', name: 'type' } })
+export class UserExperience {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @Column()
+  userId: string;
+
+  @ManyToOne(() => User, (user) => user.experiences)
```

Comment:
for relentionships use `import type { User }` and string for entity:
```suggestion
  @ManyToOne('User', (user) => user.experiences)
```

it helps with dependency cycles, please update other places for new entities